gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -759,6 +758,19 @@ private void cancelCurrentRequest() {
     contentChannelEndOffset = -1;
   }
 
+  private void drainIterator() {
```

Comment:
I think we should add a comment here why we are doing this which should ideally be handled by the client library.

Reply:
Good call. Updating